.travis.yml

```diff
@@ -9,6 +9,17 @@ install:
 # command to run tests
 script:
   - ./run_tests.sh all
+jobs:
+  include:
+  - stage: format-check
```

Comment:
I feel this should be called `lint` instead. What if we want to run some more checks in parallel in future? (pyflakes in particular comes to mind.)

Reply:
pyflakes is more of a linter. This doesn't check for any kind of correctness, it just formats. Thus, format-check.

Reply:
Sure, but I don't think we should have any of these sorts of checks block any of the others from running in CI.

Reply:
Doesn't this run last?

Reply:
Yeah, but I'm suggesting we might want to run pyflakes alongside black in future. Or maybe even now actually. Seems like a good idea at least.